airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -93,13 +95,22 @@
   public static final String DEFAULT_NETWORK = "host";
 
   private final Function<String, String> getEnv;
+  private LogConfiguration logConfiguration;
 
   public EnvConfigs() {
     this(System::getenv);
   }
 
   EnvConfigs(final Function<String, String> getEnv) {
     this.getEnv = getEnv;
+    this.logConfiguration = new LogConfiguration(
```

Comment:
Delegate results in a ton of duplicate parallel code and difficulties setting stuff up for testing.  Using a plain old pojo provides the same guarantee of a single source of truth, without all the extra boiilerplate and noise.  It integrates more easily with other parts of the codebase.  It makes code predictable and not side-effecting, and better able to run in isolation (compared to accessing environment variables from everywhere, which hides context from developers trying to reason about a black-box api call).  It makes state explicit rather than implicit, which is simpler to trace, understand, and control.